content/pages/features/reader.html

```diff
@@ -0,0 +1,229 @@
+{#
+  The goal of this page is to highlight reader features.
+#}
+<html>
+  <head>
+    <title>Documentation reader features</title>
+    <meta name="template" content="page" />
+    <meta name="slug" content="features/reader" />
+    {# <meta name="status" content="hidden" /> #}
+    <meta name="description" content="Give your documentation readers a number of in-documentation features that they'll love. Powered by Read the Docs APIs and Javascript integrations." />
+  </head>
+  <body>
+
+{#
+  View this page at http://localhost:8080/features/reader/
+#}
+
+{% import "partials/basic.html" as basic %}
+{% import "partials/features.html" as about %}
+{% import "partials/marketing.html" as marketing %}
+{% import "partials/technical.html" as technical %}
+{% import "partials/homepage.html" as homepage %}
+
+<section>
+  <div class="ui horizontally very padded container">
+    <div class="ui very padded centered grid">
+      <div class="middle aligned row">
+
+        <div class="ten wide computer sixteen wide tablet sixteen wide mobile column">
+          <div class="ui large left aligned basic vertical segment">
+            <h1 class="ui huge header">
+                Give your documentation readers a great experience
+            </h1>
+
+            <p>
+              Documentation UX is as important as the content itself.
+              Give your readers a number of in-documentation features that they'll love.
+
+            </p>
+
+            {# This should be a list of benefits, not features, since we have a feature listing in the side nav #}
+            <div class="ui list">
+              <div class="item"><i class="fad fa-check secondary icon"></i>Easily find content with great search</div>
```

Comment:
```suggestion
              <div class="item"><i class="fad fa-check secondary icon"></i>Find content with great search</div>
```

Not sure where we ended up on using "easily" on these pages.